From df61bfa4f77ddf0e6584b5884ed8870a5151b3a1 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sat, 14 May 2022 23:15:20 +0800
Subject: [PATCH 55/58] Revert "drm/i915/gem: add missing boundary check in
 vm_access"

This reverts commit 89ddcc81914ab58cc203acc844f27d55ada8ec0e.
---
 drivers/gpu/drm/i915/gem/i915_gem_mman.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/gem/i915_gem_mman.c b/drivers/gpu/drm/i915/gem/i915_gem_mman.c
index 92dd65befbcb8..5754bccff4d15 100644
--- a/drivers/gpu/drm/i915/gem/i915_gem_mman.c
+++ b/drivers/gpu/drm/i915/gem/i915_gem_mman.c
@@ -423,7 +423,7 @@ vm_access(struct vm_area_struct *area, unsigned long addr,
 		return -EACCES;
 
 	addr -= area->vm_start;
-	if (range_overflows_t(u64, addr, len, obj->base.size))
+	if (addr >= obj->base.size)
 		return -EINVAL;
 
 	/* As this is primarily for debugging, let's focus on simplicity */
-- 
2.27.0

